app/decorators/grant_decorator.rb

```diff
@@ -3,12 +3,55 @@
 class GrantDecorator < Draper::Decorator
   delegate_all
 
```

Comment:
add initialize method that determines if the user is signed in.

Reply:
Addressed in Grants::HomeDecorator